warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ContentIndexingColumnBasedHandler.java

```diff
@@ -293,6 +303,13 @@ protected void createEventColumn(RawRecordContainer event, Multimap<String,Norma
                         throw new RuntimeException(ex);
                     }
                 }
+                
+                boolean indexListField = contentHelper.isIndexListField(indexedFieldName);
+                boolean reverseIndexListField = contentHelper.isReverseIndexListField(indexedFieldName);
+                
+                if ((createGlobalIndexTerms && indexListField) || (createGlobalReverseIndexTerms && reverseIndexListField)) {
+                    indexListEntries(nci, indexField, reverseIndexField, reporter);
```

Comment:
indexListEntries(nci, (createGlobalIndexTerms && indexListField), (createGlobalReverseIndexTerms && reverseIndexListField), reporter).
While you are at it, you can fix lines 297 and 300 where indexField and reverseIndexField should be anded with the createGlobalIndexTerms and createGlobalReverseIndexTerms.

Reply:
I caught this while I was testing on the "other" system. It is not caught by the unit tests because the unit tests call indexListEntries directly and not through the getShardNamesAndValues method. There was no unit test at all before this. It's a start. More extensive unit testing would be warranted. 

The fix and the suggestion of combining the booleans into the call (and removing the surrounding if), for this line and lines 297 & 300, will be in the next commit.